sharkfuser/tests/test_conv_node.cpp

```diff
@@ -785,7 +785,7 @@ TEST_CASE("ConvWGradNode postValidateNode dimension validation",
           "[conv_wgrad_node]") {
   Context ctx;
   ConvWGradAttr attr;
-  int64_t n = 16, c = 128, h = 64, w = 32, k = 256, r = 1, s = 1;
+  int64_t n = 16, c = 256, h = 64, w = 32, k = 128, r = 1, s = 1;
```

Comment:
Note: the previous values leaded to the exception `ConvWGrad input channels must be divisible by the filter channels` added by me for `group` validation due to invalid weight shape `[c, k, r, s]`:
```
xT->getDim()[channelsIdx] % dwT->getDim()[channelsIdx] != 0;
c % k != 0 // will be 128 != 0 here
```
To validate the exact exception `ConvWGrad output DW dimensions do not match the expected shapes inferred based on input dimensions` here I replaced values `k` and `c`.